### ISSUE TITLE
[audit] Tooltip audit

### ISSUE THREAD

**Issue by jonbell-lot23** [replayio/design#14]
We do a pretty good job with this basic accessibility feature of HTML but we're missing it in some spots.

**Comment by gideonred** [comment]
This ticket has been moved to [DES-23](https://linear.app/replay/issue/DES-23/[audit]-tooltip-audit)
